sendgrid_backend/mail.py

```diff
@@ -40,6 +40,15 @@ def __init__(self, *args, **kwargs):
             raise ImproperlyConfigured("settings.py must contain a value for SENDGRID_API_KEY.  " +
                                        "You may also pass a value to the api_key argument (optional).")
 
+        sandbox_mode_in_debug = True
+        if hasattr(settings, "SENDGRID_SANDBOX_MODE_IN_DEBUG"):
+            sandbox_mode_in_debug = settings.SENDGRID_SANDBOX_MODE_IN_DEBUG
+
+        if settings.DEBUG and sandbox_mode_in_debug:
+            self.sandbox_mode = True
```

Comment:
self.sandbox_mode = settings.DEBUG and sandbox_mode_in_debug

Reply:
Ah of course! Thanks